docs/source/releasenotes.rst

```diff
@@ -5,6 +5,9 @@ Release notes
 `Upcoming release <https://github.com/robocorp/rpaframework/projects/3#column-16713994>`_
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
+- Library RPA.Assistant:
+    - Bugfixes for dialog clearing
+    - ``Add Slider`` keyword to create sliders for numeric inputs
```

Comment:
```suggestion
- Library **RPA.Assistant** (:pr:`830`; ``rpaframework-assistant`` **1.1.0**):
    - Bugfixes for dialog clearing.
    - New ``Add Slider`` keyword to create sliders for numeric inputs.
```
And is better to use instead of `:pr:` directive the issue one:
```
:issue:`<nr>`
```
Once you create and link an Issue to this PR. (so we can track them down in the release notes -- see examples from past releases below)

Reply:
great points, thanks! fixed